README.md

```diff
@@ -26,10 +36,16 @@ The Python dependencies are listed in requirements.txt are best satisfied using
 
 ## Installation
 
+Ensure you have one of the following quantum chemistry packages installed:
+* ORCA (v4.0+), Gaussian (G09 or G16), NWChem (v6.6+), QChem (v5.4+)
+* For semi-empirical calculations: XTB (v6.1+) or MOPAC (2016)
+
 To install **autodE** with [conda](https://anaconda.org/conda-forge/autode):
 ```
 conda install autode -c conda-forge
 ```
+autodE requires **Python 3.7** or higher and can be installed either via Conda or directly from the source. It is compatible with various quantum chemistry software packages.
```

Comment:
We actually [require](https://github.com/duartegroup/autodE/blob/1075b84d22e1bcc0ceba69152305ce155aed4494/setup.py#L36) at least Python 3.8. I think the minimum Python version is definitely useful information though, and worth including in the README